### PR TITLE
tiup: add 4 docs dm-edit config/help/scale in/scale out

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add 4 docs:
- tiup/tiup-component-dm-edit-config.md
- tiup/tiup-component-dm-help.md
- tiup/tiup-component-dm-scale-in.md
- tiup/tiup-component-dm-scale-out.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Joyinqin** [comment]
waiting for https://github.com/pingcap/docs/pull/5270 to merge to fix the dead link

File: tiup/tiup-component-dm-edit-config.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+> + When you modify the configuration, you cannot add or delete machines. For how to add machines, see [Scale out a cluster](/tiup/tiup-component-dm-scale-out.md). For how to delete machines, see [Scale in a cluster](/tiup/tiup-component-dm-scale-in.md).
+> + After you execute the `tiup dm edit-config` command, the configuration is modified only on the control machine. Then you need to execute the `tiup dm relaod` command to reload the configuration.
+
+## Syntax
+
+```shell
+tiup dm edit-config <cluster-name> [flags]
+```
+
+`<cluster-name>`: the cluster to operate on
```
Comment (ran-huang) [inline tiup/tiup-component-dm-edit-config.md:20]:
```suggestion
`<cluster-name>`: the cluster to operate on.
```

File: tiup/tiup-component-dm-edit-config.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+### -h, --help
+
+- Prints the help information.
+- Data type: `BOOLEAN`
+- Default: false
+
+## Output
+
+- Normally, no output.
+- If you have mistakenly modified the fields that cannot be modified, when you save the file, an error is reported, reminding you to edit the file again. For the fields that cannot be modified, see [topology file](/tiup/tiup-dm-topology-reference.md).
```
Comment (ran-huang) [inline tiup/tiup-component-dm-edit-config.md:33]:
```suggestion
- If you have mistakenly modified the fields that cannot be modified, when you save the file, an error is reported, reminding you to edit the file again. For the fields that cannot be modified, see [the topology file](/tiup/tiup-dm-topology-reference.md).
```

File: tiup/tiup-component-dm-scale-out.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+The `tiup dm scale-out` command is used for scaling out the cluster. The internal logic of scaling out the cluster is similar to the cluster deployment. The `tiup-dm` components first establish an SSH connection to the new node, create the necessary directories on the target node, then perform the deployment and start the service.
+
+## Syntax
+
+```shell
+tiup dm scale-out <cluster-name> <topology.yaml> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it with the [cluster list](/tiup/tiup-component-dm-list.md) command.
+`<topology.yaml>`: the prepared [topology file](/tiup/tiup-dm-topology-reference.md).
```
Comment (ran-huang) [inline tiup/tiup-component-dm-scale-out.md:16]:
```suggestion
`<topology.yaml>`: the prepared [topology file](/tiup/tiup-dm-topology-reference.md). This topology file should only contain the new nodes that are to be added to the current cluster.
```
@lucklove PTAL if this is correct. 
Chinese text: `<topology.yaml> 为事先编写好的扩容拓扑文件，该文件应当仅包含扩容部分的拓扑`.

File: tiup/tiup-component-dm-scale-out.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+### -i, --identity_file
+
+- Specifies the key file used to connect to the target machine.
+- Data type: `STRING`
+- If this option is not specified in the command, the `~/.ssh/id_rsa` file is used to connect to the target machine by default.
+
+### -p, --password
+
+- Specifies the password used to connect to the target machine. Do not use this option with `-i/--identity_file` at the same time.
```
Comment (ran-huang) [inline tiup/tiup-component-dm-scale-out.md:34]:
```suggestion
- Specifies the password used to connect to the target machine. Do not use this option and `-i/--identity_file` at the same time.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Pull request merged**